src/core/services/LabService.ts

```diff
@@ -92,7 +103,28 @@ export class LabServiceImpl implements LabService {
           pricePerShare: toBN('1')
             .times(10 ** backscratcherData.decimals)
             .toFixed(0),
-          apy: backscratcherData.apy,
+          apy: backscratcherData.apy
+            ? {
+                ...backscratcherData.apy,
+                composite: backscratcherData.apy.composite
```

Comment:
What is all this? it would be helpful if the logic was in named constants

Reply:
This is due to having different naming inside the apy data. Fibo did fix it on yearn-exporter to avoid this, but the PR was not accepted because it was decided not to change yearn-exporter.